docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -45,14 +47,11 @@ Variables are part of a process instance and represent the data of the instance.
 
 ### Implement a form
 
-1. To add a form and decide what's for dinner, return to the **Modeler** homepage and click **New > Form**.
+1. To add a form and decide what's for dinner, navigate to the **Modeler** Home page and click on the project you are working on. Then click **New > Form**.
```

Comment:
This should read "Navigate to the Modeler **Home** page and click the project you are working on. Then, click **New > Form**."